src/saga/adaptors/lsf_summit/lsfjob_summit.py

```diff
@@ -0,0 +1,1068 @@
+    else:
+        env_variable_list = ""
+
+    # a workaround is to do an explicit 'cd'
+    if jd.working_directory is not None:
+        lsf_params += "#BSUB -cwd %s \n" % jd.working_directory
+
+    if jd.output is not None:
+        # if working directory is set, we want stdout to end up in
+        # the working directory as well, unless it containes a specific
+        # path name.
+        if jd.working_directory is not None:
+            if os.path.isabs(jd.output):
+                lsf_params += "#BSUB -o %s \n" % jd.output
+            else:
+                # user provided a relative path for STDOUT. in this case 
+                # we prepend the working directory path before passing
+                # it on to LSF.
+                lsf_params += "#BSUB -o %s/%s \n" % (jd.working_directory, jd.output)
+        else:
+            lsf_params += "#BSUB -o %s \n" % jd.output
+
+    if jd.error is not None:
+        # if working directory is set, we want stderr to end up in 
+        # the working directory as well, unless it contains a specific
+        # path name. 
+        if jd.working_directory is not None:
+            if os.path.isabs(jd.error):
+                lsf_params += "#BSUB -e %s \n" % jd.error
+            else:
+                # user provided a relative path for STDERR. in this case
+                # we prepend the working directory path before passing
+                # it on to LSF.
+                lsf_params += "#BSUB -e %s/%s \n" % (jd.working_directory, jd.error)
+        else:
+            lsf_params += "#BSUB -e %s \n" % jd.error
+
+
+    if jd.wall_time_limit is not None:
+        hours = jd.wall_time_limit / 60
+        minutes = jd.wall_time_limit % 60
+        lsf_params += "#BSUB -W %s:%s \n" \
+            % (str(hours), str(minutes))
+
+    if (jd.queue is not None) and (queue is not None):
+        lsf_params += "#BSUB -q %s \n" % queue
+    elif (jd.queue is not None) and (queue is None):
+        lsf_params += "#BSUB -q %s \n" % jd.queue
+    elif (jd.queue is None) and (queue is not None):
+        lsf_params += "#BSUB -q %s \n" % queue
+
+    if jd.project is not None and ':' in jd.project:
+        account, reservation = jd.project.split(':',1)
+        lsf_params += "#BSUB -P %s \n" % str(account)
+        lsf_params += "#BSUB -U %s \n" % str(reservation)
+    elif jd.project is not None:
+        lsf_params += "#BSUB -P %s \n" % str(jd.project)
+
+    if jd.job_contact is not None:
+        lsf_params += "#BSUB -u %s \n" % str(jd.job_contact)
+
+    # Request enough nodes to cater for the number of cores requested
+    if jd.total_cpu_count is None:
+        total_cpu_count = 1
+    else:
+        total_cpu_count = jd.total_cpu_count
+
+    ppn = 42
+    number_of_nodes = int(total_cpu_count / ppn)
+    if total_cpu_count % ppn > 0:
+        number_of_nodes += 1
+    lsf_params += "#BSUB -nnodes %s \n" %str(number_of_nodes)
+
+    lsf_params += "#BSUB -alloc_flags 'gpumps smt4' \n"
+
```

Comment:
The 5 lines above are the only change really compared to the original adaptor, right?  If that's the case, we should `ifdef` that part and merge back to the original adaptor as soon as it works...

Reply:
As soon as @vivek-bala says it submits jobs correctly, I will put it in the original with a `if summit` statement.